docker-compose.yml

```diff
@@ -6,7 +6,8 @@ services:
       dockerfile: Dockerfile
     container_name: worklenz_frontend
     ports:
-      - "4200:4200"
+      - "80:80"
+      - "44:443"
```

Comment:
Your nginx configuration is not using port 443. Maybe you miss the nginx configuration for https? If this is not the case, remove the unused port.